### ISSUE TITLE
failed to remove original file: process cannot access the file ebcause it is being used by another process

### ISSUE THREAD

**Issue by mikesTech** [majd/ipatool#248]
### What happened?

linux doesnt complain as much as windows, so even though it was allegedly fixed before, it wasnt really on windows
i compiled it myself, didnt use the release
i had to fix it myself by adding tmpFile.Close() where zipreader and writer were closed in appstore_replicate_sinf (line 72 around there)

### Version

latest

### Relevant log output

```shell
D:\Users\\Desktop\ipatool-main>ipatool download -b com.hammerandchisel.discord -o discord2.ipa
waiting 8 Seconds1:51PM ERR error="failed to rename original file: rename discord2.ipa.tmp discord2.ipa: The process cannot access the file because it is being used by another process." success=false
```

**Comment by mikesTech** [comment]
please check my pull request

**Comment by majd** [comment]
This will be fixed once #249 is merged.
